cypress/support/step_definitions/dashboard/i_am_logged_in_as_a_string.js

```diff
@@ -1,14 +1,6 @@
 /* eslint-disable */
 Given("I am logged in as a {string}", (role) => {
   const roles = role === "Administrator" ? { roles: [role] } : { roles: [] };
-  cy.task("db", {
```

Comment:
Deleting is used so we can essentially reset the state for this user. Ideally you'd setup and teardown per test because you don't want another test file to carry over any state and affect the current test you're in. Is this affecting your tests?